functions-and-scope.js

```diff
@@ -72,6 +128,19 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+//Stap 1: Maak nieuwe lijst waarin hoogste getal wordt opgeslagen
+const highestGradeList = [0];
```

Comment:
Het is beter om hier een let van te maken en de eerste waarde uit de array te pakken:
`let highestGrade = grades[0];`